reviews/source_control/models.py

```diff
@@ -22,3 +24,50 @@ class PullRequest:
     approved: str
     approved_by_others: bool
     labels: List[Label] = field(default_factory=list)
+
+    def render_approved(self) -> str:
+        """Renders the approved status as a colourised string"""
+        if self.approved == "APPROVED":
+            return "[green]Approved[/]"
+        if self.approved == "CHANGES_REQUESTED":
+            return "[red]Changes Requested[/]"
+        return ""
+
+    def render_approved_by_others(self) -> str:
+        """Renders approved_by_others flag as a colourised string"""
+        return "[green]Ready[/]" if self.approved_by_others else ""
+
+    def render_labels(self, colour_map: Dict[str, str]) -> str:
+        """Renders the labels as a joined colourised string"""
+        return ", ".join(
+            [
+                f"{colour_map.get(label.name.lower(), '[white]')}{label.name}"
+                for label in self.labels
+            ]
+        )
+
+    def render_title(self, org: str, repository: str) -> str:
+        """Renders the title as a colourised string"""
+        colour, title = "", self.title
+
+        if title.startswith("[Security]"):
+            title = title.removeprefix("[Security]")
+            colour = "[bold red][Security][/]"
+        elif self.draft:
```

Comment:
A draft PR could also be a security PR which might be nice if it rendered as `[Security][Draft] an example pull request` until it's promoted to a pull request and becomes  `[Security] an example pull request`. 

what do you think? do you have a preference?

```suggestion
        if self.draft:
```

Reply:
makes sense. In this case, would we want to keep the colours as `red` and `gray` in the title?

i.e. `[bold red][Security][/][bold grey][Draft] [/]`

Reply:
yeah, I think keeping the red and grey makes sense here as it keeps it consistent with how the tags will be used individually 👍